tests/test_notifications.py

```diff
@@ -262,7 +261,7 @@ def test_node_subscriptions_and_backrefs_removed_when_node_is_deleted(self):
             subscription.save()
 
             s = getattr(project.creator, 'email_transactional', [])
-            assert_equal(len(s), 1)
+            assert_equal(len(s), 2)
```

Comment:
While we're in [this function](https://github.com/CenterForOpenScience/osf.io/blob/711558a79268f0b72630083b3c2a70d97875fe9e/tests/test_notifications.py#L253-L275), let's clean up the indentation (it should be dedented four spaces).